packages/webui/src/client/ui/UserEditOperations/RenderUserEditOperations.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { assertNever, clone } from '@sofie-automation/corelib/dist/lib'
+import { RundownId } from '@sofie-automation/corelib/dist/dataModel/Ids'
+import { CoreUserEditingDefinition } from '@sofie-automation/corelib/dist/dataModel/UserEditingDefinitions'
+import { JSONBlobParse, UserEditingType, UserOperationTarget } from '@sofie-automation/blueprints-integration'
+import { translateMessage } from '@sofie-automation/corelib/dist/TranslatableMessage'
+import { MenuItem } from '@jstarpl/react-contextmenu'
+import { i18nTranslator } from '../i18n'
+import { doModalDialog } from '../../lib/ModalDialog'
+import { SchemaFormInPlace } from '../../lib/forms/SchemaFormInPlace'
+import { doUserAction, UserAction } from '../../lib/clientUserAction'
+import { MeteorCall } from '../../lib/meteorApi'
+
+export function RenderUserEditOperations(
+	isFormEditable: boolean,
+	rundownId: RundownId,
+	targetName: string,
+	userEditOperations: CoreUserEditingDefinition[] | undefined,
+	operationTarget: UserOperationTarget
+): React.JSX.Element | null {
```

Comment:
This looks like a regular React component, but isn't implemented as one. Is there a particular reason?

Reply:
:shrug: It's been changed now